tests/integration/integrations/kubeflow/orchestrators/test_kubeflow_orchestrator.py

```diff
@@ -104,7 +104,7 @@ def test_kubeflow_orchestrator_remote_stack(
 
 
 @pytest.mark.skipif(
-    sys.version_info > (3, 10),
+    sys.version_info >= (3, 11),
     reason="Kubeflow integration not installed in Python 3.11",
```

Comment:
why don't we install kubeflow in python 3.11?

Reply:
fixed now should be fine, i think we need to visit all tests that are being skipped

Reply:
Yep. Can you create a ticket to revisit skipped tests and TODOs where dependencies are pinned?